providers/aws/resources/aws.lr.manifest.yaml

```diff
@@ -898,6 +898,8 @@ resources:
         Use the `aws.ec2` resource to assess the configuration of AWS EC2 instances.
     fields:
       ebsEncryptionByDefault: {}
+      eips:
+        min_mondoo_version: 9.0.0
```

Comment:
Shouldn't this be 11+?

Reply:
bc providers auto update / aren't tied to the version, anything after 9 should work

Reply:
Ah, good to know. I wasn't aware of this. Thanks.